### DISCUSSION TITLE
Support GIF NFTs in portfolio view

### DISCUSSION THREAD

**Discussion by wxivd** [asastats/channel#97] [Ideas]
Whereas the primary function of displaying owned NFTs is price information and valuation, there is another aspect worthy of consideration and that is aesthetic presentation. 

I suggest adding support for Gif NFTs. What are your thoughts on this.

**Comment by ipaleka** [comment]
You mean animated GIFs?

That would inevitably lead to calls for having videos, readable pdfs, ...

P.S. and my opinion is that we should leave that aspect to the user widgets providers...

**Reply by wxivd** [reply]
> You mean animated GIFs?

Animated NFTs Similar to these

[SoulPod](https://www.nftexplorer.app/collection/soulpod)
[AlgoMeteo](https://www.nftexplorer.app/collection/algometeo)
[EncryptedPunks](https://www.nftexplorer.app/collection/encryptedpunks) 
[AlgoFauna](https://www.nftexplorer.app/collection/algofauna) 

> That would inevitably lead to calls for having videos, readable pdfs,

We can apply same logic to pictures. that would inevitably lead to calls for animation :)

If the NFT contains animation, sound or short video then the NFT viewer should be able to present that. Actually I see it more of a bug that needs to be fixed than an advanced feature. This is basic function, requesting accurate presentation of NFTs

> P.S. and my opinion is that we should leave that aspect to the user widgets providers...

Widgets provide advanced features that a user would be incentivised to pay for. Why would a user pay for this if it is a standard at Rand Gallery, AGOxNFT, NFTexplorer

**Reply by ipaleka** [reply]
> > That would inevitably lead to calls for having videos, readable pdfs,
> 
> We can apply same logic to pictures. that would inevitably lead to calls for animation :)

That's a fallacy.

An image of an NFT is a humanized presentation of the self-sufficient ID, name, and unit. We deliberately choose to present them as image as that in 99% of cases represents a correct visual **identification** for the users.

> If the NFT contains animation, sound or short video then the NFT viewer should be able to present that. Actually I see it more of a bug that needs to be fixed than an advanced feature. This is basic function, requesting accurate presentation of NFTs

The only accurate presentation of NFTs implies serving the **original** NFTs. And that would demand resources about a thousand times bigger than ours, users bandwidths hundreds of times bigger than the existing and speed ten to hundred times slower for our users.

So we decide to fake and we base that faking primarily on a performance perspective. And we provide absolute accuracy for users by clicking the image.

> Widgets provide advanced features that a user would be incentivised to pay for. Why would a user pay for this if it is a standard at Rand Gallery, AGOxNFT, NFTexplorer

That's not true, it is expected that the vast majority (I'm refraining from saying 90%...) of user widgets will be free for use.

They will be free because for the majority of those simple widgets their development would be basically a one-time effort with a little of maintaining in the future. Also, we'd be sponsoring the development of those *basic* sets of user widgets that users will be able to select on their settings page.

Presenting an original image, video, audio, or PDF of an NFT is surely something we should have in mind when we would be deciding on sponsoring.
